engine/lite/embeddedDeploymentManager/src/test/scala/pl/touk/nussknacker/streaming/embedded/StreamingEmbeddedDeploymentManagerTest.scala

```diff
@@ -142,14 +143,10 @@ class StreamingEmbeddedDeploymentManagerTest extends BaseStreamingEmbeddedDeploy
     def message(input: String) = obj("message" -> fromString(input)).noSpaces
     def prefixMessage(prefix: String, message: String) = obj("message" -> fromString(message), "prefix" -> fromString(prefix))
 
-
     fixture.deployScenario(scenarioForOutput("start"))
 
-
     kafkaClient.sendMessage(inputTopic, message("1")).futureValue
-
-    val consumer = kafkaClient.createConsumer().consumeWithJson(outputTopic)
-    consumer.head shouldBe prefixMessage("start", "1")
+    kafkaClient.createConsumer().consume[Json](outputTopic).take(1).head.message() shouldBe prefixMessage("start", "1")
```

Comment:
Please do not change the behaviour of the test. It created consumer once and used it multiple times before your change